package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "3.49.0",
```

Comment:
When do we decide to go to 3.49.1 vs 3.50.0?

Reply:
It has consequences as far as what upgrade commands you need to run to update versions, but it doesn't make much difference for us since we're referencing the git commit hash anyway. We should follow semvar, but mainly the version just needs to change whenever we need to pull a new version into vets-website.